server.js

```diff
@@ -2,18 +2,27 @@ import express from "express";
 import morganLogger from "morgan";
 import querystring from "querystring";
 import axios from "axios";
+import envalid from 'envalid'
```

Comment:
Line 5 and 6 can be merged.

```suggestion
import envalid, { str, num, url } from 'envalid'
```